src/content/pages/build/taco.md

```diff
@@ -0,0 +1,213 @@
+---
+template: build-page/taco
+path: /build/taco
+title: TACo Page
+seoTitle: Threshold Taco
+seoDescription: "TACo (Threshold Access Control) enables end-to-end encrypted data sharing and communication."
+tacoInfo:
+  title: "TACo: Privacy without trust"
```

Comment:
This should be "TACo: Privacy Without Trust" to be consistent with the casing on similar pages (eg tBTC SDK)

Reply:
Addressed in [capitalize the taco info title](https://github.com/threshold-network/website/pull/113/commits/5d9ddaef765b53c13853149e099639572ae97a90)